scss/books-list.scss

```diff
@@ -0,0 +1,44 @@
+.book-cover-mini {
+  background: url(images/book-cover.png) center / contain no-repeat;
+  height: 200px;
+  width: 142px;
+}
+
+.book-title {
+  font-size: 16px;
+  font-weight: 700;
+  height: 15px;
+  margin: 12px 61px 4px 28px;
+  width: 109px;
+}
+
+.book-author {
+  color: $gray;
+  font-size: 14px;
+  height: 14px;
+  margin: 4px 61px 0 28px;
+  width: 109px;
+}
+
+.book-item {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-05;
```

Comment:
You can put this box-shadown in a scss global variable, since you are using it in different pages.